portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/ManagementBar.path

```diff
@@ -27,7 +27,7 @@
 </tr>
 <tr>
 	<td>FILTER_AND_ORDER_ACTIVE_ITEM</td>
-	<td>//div[contains(@class,'dropdown-menu')]//a[contains(text(),'${key_menuItem}')]</td>
+	<td>//div[contains(@class,'dropdown-menu')]//li[contains(@aria-selected, 'true')]//a[contains(text(),'${key_menuItem}')]</td>
```

Comment:
This path wasn't checking if the item was active. 
It was only checking if the item was present in the dropdown.
All the tests that use this path have been tested and the index is in the ticket comment.